### ISSUE TITLE
 Manual editing questions

### ISSUE THREAD

**Issue by INHYEOKSONG** [dphansti/CORAL#23]
hello.
I want to utilize Kinome map for Coral. I want to modify the tree, but I was wondering if you could share how to create the tree. Now, by understanding the code, it seems to be svg creation through tsv -> tree.rds file creation -> rendering through rds file.

**Comment by dphansti** [comment]
Yes, that seems about right?  The tree itself was first drawn in Adobe Illustrator and exported as an svg.  Then the code recolors it.  Modifying the structure of the tree would require more illustrator work.
